client/my-sites/site-settings/test/form-general.jsx

```diff
@@ -276,6 +229,130 @@ describe( 'SiteSettingsFormGeneral ', () => {
+			describe( 'support existing coming soon v1 sites that have not migrated', () => {
+				test( 'Coming soon option should be selected when a site is private and unlaunched (coming soon mode v1 by default)', () => {
+					const newProps = {
+						...testProps,
+						fields: {
+							blog_public: -1,
+						},
+						isUnlaunchedSite: true,
+						// So renderLaunchSite() runs smoothly.
+						siteDomains: [ 'test.com' ],
+					};
+
+					const { getByLabelText } = renderWithRedux( <SiteSettingsFormGeneral { ...newProps } /> );
+					const radioButton = getByLabelText( 'Coming soon', { exact: false } );
+					expect( radioButton ).toBeChecked();
+				} );
+
+				test( 'Should check private option when site is private, but not in coming soon v1 and not private and unlaunched', () => {
+					const newProps = {
+						...testProps,
+						fields: {
+							blog_public: -1,
+							wpcom_coming_soon: 0,
+						},
+					};
+
+					const { getByLabelText } = renderWithRedux( <SiteSettingsFormGeneral { ...newProps } /> );
+					const radioButtonComingSoon = getByLabelText( 'Coming soon', { exact: false } );
+					expect( radioButtonComingSoon ).not.toBeChecked();
+
+					// Check that we're not displaying the site as private
+					const radioButtonPrivate = getByLabelText( 'Private', { exact: false } );
+					expect( radioButtonPrivate ).toBeChecked();
+				} );
+
+				test( 'Coming soon option should be selected when a site still has coming soon v1 enabled', () => {
+					const newProps = {
+						...testProps,
+						fields: {
+							wpcom_coming_soon: 1,
+						},
+					};
+
+					const { getByLabelText } = renderWithRedux( <SiteSettingsFormGeneral { ...newProps } /> );
+					const radioButtonComingSoon = getByLabelText( 'Coming soon', { exact: false } );
+					expect( radioButtonComingSoon ).toBeChecked();
+
+					// Check that we're not displaying the site as private
+					const radioButtonPrivate = getByLabelText( 'Private', { exact: false } );
+					expect( radioButtonPrivate ).not.toBeChecked();
+				} );
+
+				test( 'Should check private option when site is in coming soon v1 mode but the etk plugin is disabled on atomic', () => {
+					const newProps = {
+						...testProps,
+						fields: {
+							wpcom_coming_soon: 1,
+						},
+						isAtomicAndEditingToolkitDeactivated: true,
+					};
+
+					const { getByLabelText, container } = renderWithRedux(
+						<SiteSettingsFormGeneral { ...newProps } />
+					);
+					expect(
+						container.querySelector( '.site-settings__visibility-label.is-coming-soon' )
+					).toBe( null );
+
+					// Check that we're not displaying the site as private
+					const radioButtonPrivate = getByLabelText( 'Private', { exact: false } );
+					expect( radioButtonPrivate ).toBeChecked();
+				} );
+			} );
+		} );
+
+		describe( 'Coming soon plugin availability', () => {
+			test( 'Should hide Coming Soon form element when the site is not atomic or the editing toolkit plugin is not disabled', () => {
```

Comment:
Add the word "not" to the test description, because this is testing that the Coming Soon form element is not hidden.

```suggestion
			test( 'Should not hide Coming Soon form element when the site is not atomic or the editing toolkit plugin is not disabled', () => {
```